SpatialGDK/Source/SpatialGDK/Public/Utils/InterestFactory.h

```diff
@@ -15,20 +15,24 @@ DECLARE_LOG_CATEGORY_EXTERN(LogInterestFactory, Log, All);
 
 namespace SpatialGDK
 {
-
-void GatherClientInterestDistances();
-
 class SPATIALGDK_API InterestFactory
 {
 public:
 	InterestFactory(AActor* InActor, const FClassInfo& InInfo, USpatialClassInfoManager* InClassInfoManager, USpatialPackageMapClient* InPackageMap);
 
+	static void CreateClientCheckoutRadiusConstraint(USpatialClassInfoManager* ClassInfoManager);
+
 	Worker_ComponentData CreateInterestData() const;
 	Worker_ComponentUpdate CreateInterestUpdate() const;
 
 	static Interest CreateServerWorkerInterest();
 
 private:
+	static QueryConstraint GetDefaultCheckoutRadiusConstraint();
+	static TMap<UClass*, float> GetActorTypeToRadius();
+	static TMap<float, TArray<UClass*>> DedupeDistancesAcrossActorTypes(TMap<UClass*, float> ComponentSetToRadius);
```

Comment:
Pass the map in as a const & to prevent the copy. Here and BuildNonDefaultActorCheckoutConstraints

Reply:
done, ty- also used references for the map, worked out why it wasn't working before